From: Gavin Lloyd <gavinhungry@gmail.com>
Date: Tue, 9 Apr 2024 17:38:02 -0700
Subject: Fix misplaced else in mnt_update_already_done

See 477401f0de

(cherry picked from commit 6b316e0c9f1feb61bef3bb14b1356be8fbefcc17)
---
 libmount/src/tab_update.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/libmount/src/tab_update.c b/libmount/src/tab_update.c
index 87512af..d44d190 100644
--- a/libmount/src/tab_update.c
+++ b/libmount/src/tab_update.c
@@ -982,9 +982,9 @@ int mnt_update_already_done(struct libmnt_update *upd)
 			if (mnt_optstr_get_missing(fs->user_optstr, upd->fs->user_optstr, NULL) == 0) {
 				upd->missing_options = 1;
 				DBG(UPDATE, ul_debugobj(upd, " missing options detected"));
-			}
-		} else
-			rc = 1;
+			} else
+				rc = 1;
+		}
 
 	} else if (upd->target) {
 		/* umount */
